2200034/Changyong.py

```diff
@@ -0,0 +1,35 @@
+from collections import deque
+
+for tc in range(1, 1+ int(input())):
+    #6 5 일때
+    N, M = map(int, input().split()) # N은 사람의 수 M개의 줄에 걸쳐서(사람의 관계 수를 나타냄)
+
+    #인접 리스트 생성 Key, Value 형태의 딕셔너리
+    adj = {x + 1: [] for x in range(N)}
+
+    for _ in range(M):    #5번의 주민 네트워크 
+        a, b = map(int,input().split())  #a, b에 주민 번호를 넣음
+        adj[a].append(b) #a주민의(숫자가 들어갈 거임) 정보에는 b주민하고 연결되있다는 것을 의미
+        adj[b].append(a) #반대로 b주민은 a와 연결되어있으니 서로 상대방 네트워크 번호를 삽입
+    visited = [0] * (N + 1) #0의 방문은 비워둔다.
+    q = deque()
+    cnt = 0  # 무리의 수
+
+    #전체 무리를 찾기 위해 for문으로 큐에 넣음
+    for i in range(1, N + 1):
+        #방문하지 않았을 경우는
+        if visited[i] == 0:
+            cnt += 1
+            q.append(i)     #큐에 값(주민의 번호)을 넣고
+            visited[i] == 1  #방문 표시를 1로 남긴다.
```

Comment:
bfs 활용 아주 좋습니다! 👍 
```suggestion
            visited[i] = 1  #방문 표시를 1로 남긴다.
```
`==`와 `=` 차이에 주의해주세요!